ckanext/validation/templates/scheming/form_snippets/resource_schema.html

```diff
@@ -1,8 +1,10 @@
 {% import 'macros/form.html' as form %}
 
   {% set value = data[field.field_name] %}
-  {% set is_url = value and value[4:]|lower == 'http' %}
-  {% set is_json = not is_url and value %}
+  {# % set is_url = value and value[4:]|lower == 'http' % #}
+  {# % set is_json = not is_url and value % #}
```

Comment:
Are these commented out lines still needed?

Reply:
I left it in for now because of this unresolved discussion: https://github.com/frictionlessdata/ckanext-validation/issues/92